lib/Image.php

```diff
@@ -212,7 +223,9 @@ public static function wp_upload_dir() {
 	 */
 	function init( $iid = false ) {
 		if ( !$iid ) { Helper::error_log('Initalized TimberImage without providing first parameter.'); return; }
-		
+		if ( $iid instanceof Image ) {
```

Comment:
Would `$iid instanceof __CLASS__` work here? In case we ever change the class name.